README.md

```diff
@@ -50,11 +50,10 @@ Planned features:
     (divide [this x y] (/ x y)))
 
 ;; A proxied implementation of MathService for use via RPC
-(defservice remote-math-service
+(def-remote-service remote-math-service
```

Comment:
yeah -- it's weird and unusual but `defremoteservice` also felt weird/wrong? which do you think is less wrong?